cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -1354,3 +1354,175 @@ def test_update_committee_threshold_out_of_range(
+
+        def _check_rat_enact_state(
+            action_txid: str, action_type: governance_utils.PrevGovActionIds
+        ) -> None:
+            # Check ratification
+            _cur_epoch_rat = cluster.wait_for_new_epoch(padding_seconds=5)
+
+            rat_gov_state = cluster.g_conway_governance.query.gov_state()
+            conway_common.save_gov_state(
+                gov_state=rat_gov_state, name_template=f"{temp_template}_rat_{_cur_epoch_rat}"
+            )
+
+            rat_action = governance_utils.lookup_ratified_actions(
+                gov_state=rat_gov_state, action_txid=action_txid
+            )
+            assert rat_action, "Action not found in ratified actions"
+
+            # Wait for enactment
+            _cur_epoch_enact = cluster.wait_for_new_epoch(padding_seconds=5)
+
+            # Check enactment
+            assert _cur_epoch_enact == _cur_epoch_rat + 1, f"Unexpected epoch {_cur_epoch_enact}"
+            enact_gov_state = cluster.g_conway_governance.query.gov_state()
+            conway_common.save_gov_state(
+                gov_state=enact_gov_state, name_template=f"{temp_template}_enact_{_cur_epoch_enact}"
+            )
+
+            prev_action_rec = governance_utils.get_prev_action(
+                action_type=action_type,
+                gov_state=enact_gov_state,
+            )
+
+            assert prev_action_rec.txid == action_txid, "Action not enacted"
+
+        deposit_amt = cluster.conway_genesis["govActionDeposit"]
+        anchor_url = "http://www.cc-update.com"
+        anchor_data_hash = "5d372dca1a4cc90d7d16d966c48270e33e3aa0abcb0e78f0d5ca7ff330d2245d"
+        prev_action_rec = governance_utils.get_prev_action(
+            action_type=governance_utils.PrevGovActionIds.COMMITTEE,
+            gov_state=cluster.g_conway_governance.query.gov_state(),
+        )
+
+        # Set CC threshold to zero
+        update_threshold_action = cluster.g_conway_governance.action.update_committee(
+            action_name=temp_template,
+            deposit_amt=deposit_amt,
+            anchor_url=anchor_url,
+            anchor_data_hash=anchor_data_hash,
+            threshold="0",
+            prev_action_txid=prev_action_rec.txid,
+            prev_action_ix=prev_action_rec.ix,
+            deposit_return_stake_vkey_file=pool_user_lg.stake.vkey_file,
+        )
+
+        tx_files = clusterlib.TxFiles(
+            proposal_files=[update_threshold_action.action_file],
+            signing_key_files=[
+                pool_user_lg.payment.skey_file,
+            ],
+        )
+
+        tx_output_action = clusterlib_utils.build_and_submit_tx(
+            cluster_obj=cluster,
+            name_template=f"{temp_template}_update_threshold",
+            src_address=pool_user_lg.payment.address,
+            use_build_cmd=True,
+            tx_files=tx_files,
+        )
+
+        action_txid = cluster.g_transaction.get_txid(tx_body_file=tx_output_action.out_file)
```

Comment:
The `action_txid` and `action_ix`, as well as the other variables corresponding to the "threshold" action should be renamed to something more specific, as this is not the only action we have in this test (the other being "const_action").